components/application/logout.jsx

```diff
@@ -7,7 +7,7 @@ import { withGlobalStore } from 'store'
 
 const LOGOUT_URL = `https://api.dev.core.ac.uk/logout?continue=${encodeURI(
```

Comment:
Perhaps, you meant [`encodeURIComponent`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent)